doc/problem/behavior.md

```diff
@@ -1,31 +1,38 @@
+1. how they drink, feed or move (locomotion) and
+2. how they translate between active state or inactive state.
 
-## Background Knowledge:
-- Mouses react differently during Active state and Inactive states and all behavioral record should be classified into 2 mutually exclusive categories, Active States (ASs) and Inactive States (ISs). To designate ISs, we examined all time intervals occurring between movement, feeding, and drinking events while the animal was outside the Home base. Those time intervals exceeding an IS Threshold (IST) duration value were classified as ISs; the set of ASs was then defined as the complement of these ISs. Equivalent mathematically, ASs can also be defined as those intervals resulting from connecting gaps between events outside the Home base of length at most IST; ISs are then defined as the complement of these ASs. (*Active State Organization of Spontaneous Behavior Patterns*, C. Hillar et al.)
+From the paper we have the following key background information from the paper:
 
+>Mouses react differently during Active state and Inactive states and all behavioral record should be classified into 2 mutually exclusive categories, Active States (ASs) and Inactive States (ISs). To designate ISs, we examined all time intervals occurring between movement, feeding, and drinking events while the animal was outside the Home base. Those time intervals exceeding an IS Threshold (IST) duration value were classified as ISs; the set of ASs was then defined as the complement of these ISs. Equivalent mathematically, ASs can also be defined as those intervals resulting from connecting gaps between events outside the Home base of length at most IST; ISs are then defined as the complement of these ASs. (*Active State Organization of Spontaneous Behavior Patterns*, C. Hillar et al.)
 
-## Behavioral Profile:
 
-For In/Active State:
-- AS Probability: the probability of AS among all time period 
-- AS Duration: the length of time of AS
-- AS Rate: the reciprocal of average length of time of AS 
+## Statement of statistical problems (including how and why it maps to the Overall Problem Statement):
 
-Events: Drinking, Feeding and Locomotion
-- AS Event Intensity: the average consumption size or distance in one minute of AS
-- Event size, Event rate: the number and frequency of events in AS time
+The above flowchart shows the key metrics that are required by the study to capture the behavioural profile:
+
+![alt tag](http://cenzhuoyao.com/wp-content/uploads/2016/04/project1_behavior_profile.png)
 
-Bouts: split all the time duration to several event bout
-- Bout size, Bout routine: the number and frequency of bouts in one event
-- Bout duration, Bout density: the length of time and frequency of bout 
+The main focus is 3 key states of the mice i.e. [_Drinking | Feeding | Locomotion_]
+
+- [_Drinking | Feeding | Locomotion_]
+    - AS [_Drinking | Feeding | Locomotion_] Intensity
```

Comment:
A note about intensity: We are not entirely sure what the Tecott Lab's meaning of "intensity" is. Our current hypothesis is that intensity is defined as quantity over active state time. E.g. for drinking, intensity is the quantity consumed divided by the total amount of time the mouse is in an active state.